src/modules/auth/redux/consts.ts

```diff
@@ -0,0 +1,37 @@
+export const actionConsts = {
+  auth: {
+    setCurrentUser: 'AUTH.SET_CURRENT_USER',
+  },
+};
+
+/*
+  Prepare: initializing auth state (From storage/cookie)
+  Pending: submitting authorization request and pending response
+  Authorized: authorized
+  Unauthorized: unauthorized
+  NeedRefresh: the token has been expired, and has to re-authorize
+*/
+
+export enum AuthState {
+  Prepare = 'AuthState.Prepare',
+  Pending = 'AuthState.Pending',
+  Authorized = 'AuthState.Authorized',
+  Unauthorized = 'AuthState.Unauthorized',
+
+  NeedRefresh = 'AuthState.NeedRefresh',
+}
+
+export enum AuthConsts {
+  SET_AUTH_STATE = 'AuthStore_SetAuthState',
+  SET_AUTH_DATA = 'AuthStore_SetAuthData',
+  SET_CURRENT_USER = 'AuthStore_SetCurrentUser'
+}
+
+export const INITIAL_STATE = {
+  currentUser: {}, 
+  state: AuthState.Unauthorized,
+  data: {}, 
+};
+
+export const STORE_KEY = 'authStore';
```

Comment:
`STORE_KEY` may be confused with root state, maybe change another name for authStore?

Reply:
this stands for the property name to be set in redux state store for the auth module.